coding-concepts.js

```diff
@@ -62,7 +70,23 @@ class Learn {
   }
 }
 const learnStudent = new Learn("George")
-// console.log(learnStudent)
-
-// a) Your answer:
-// b) Verify and explain:
+console.log(learnStudent)
+
+// a) Your answer: {student: George,
+                  // cohort: "foxtrot",
+                  // year: 2022
+                  // }
+// b) Verify and explain: Learn { student: 'George', cohort: 'Foxtrot', year: 2022 } Almost correct, I wasn't expecting the Learn at the beginning. A class is a blueprint for an object. In this case the class creates a new instance of a Learn student object who is in foxtrot in 2022. The Constructor method has a paramater of name which expects a students name to be passed through when creating a new instance of the class. In this way you could quickly create a lot of students who are also in foxtrot 22.
+
+//The below was part of my testing for an answer to question 2 of the interview questions
+let map = Array.prototype.map
```

Comment:
Nice job in this section!  I really like that you extend your research in this section as well!👏